### PR TITLE
Add missing documentation for TreeCtrl GetSpacing / SetSpacing

### PR DESCRIPTION
**Pull request by swt2c**

### CODE REVIEW

File: interface/wx/treectrl.h [hunk 1]
```diff
@@ -506,6 +506,12 @@ class wxTreeCtrl : public wxControl
     */
     virtual unsigned int GetIndent() const;
 
+    /**
+        Returns the current tree control spacing (the number of pixels between
+        the start and the Text).  Has no effect under wxMSW.
```
Comment (vadz) [inline interface/wx/treectrl.h:511]: Sorry, this doesn't seem quite correct to me:

1. Start of what? And why is the "Text" capitalized?
1. "Has no effect" doesn't make much sense for the accessor IMO (although it's OK for the setter).

**Review by vadz — COMMENTED** [review]
Thanks, they should definitely be documented, but the description looks a bit strange to me, would you mind improving it a bit?

FWIW I think we should also initialize `m_spacing` to 0 in the base class ctor (as it's not used anyhow in wxMSW version) and use `FromDIP()` in the generic code instead of hardcoding 18 and 10 values for it...

File: interface/wx/treectrl.h (hunk 1 above)
Comment (swt2c) [reply interface/wx/treectrl.h]: Okay, so I removed "has no effect" from the accessor and removed the capitalization of Text.  Other than that, I'm not entirely sure how to improve the description.  I basically copied it from the include/wx/treectrl.h header comments.  Personally, I don't actually understand what 'Spacing' means exactly in this particular case.  I only ran into this when porting something to wxPython 4.0 and these functions were missing.

Comment (vadz) [reply interface/wx/treectrl.h]: To be honest, I'm not too sure myself, but the treectrl sample has menu items to increase/decrease it, so it should be visible in it, I guess? From looking at the code it looks like it's the space before the actual text reserved for showing the button/checkbox before the item.

Comment (swt2c) [reply interface/wx/treectrl.h]: Okay, I played around a bit with the sample to see what it does.  See what you think of my new description.

**Comment by vadz** [comment]
Thanks a lot, and sorry for my nitpicking!

**Comment by swt2c** [comment]
Thanks, and do you mind cherry-picking this to WX_3_0_BRANCH also?  In case Robin ever does another 4.0.x release.  Thanks.

**Pull request closed without merging**